gusto/time_discretisation/time_discretisation.py

```diff
@@ -101,6 +101,10 @@ def __init__(self, domain, field_name=None, solver_parameters=None,
             elif self.wrapper_name == "recovered":
                 self.wrapper = RecoveryWrapper(self, options)
             elif self.wrapper_name == "supg":
+                if options.field_names:
```

Comment:
Could this be `if options.field_names is not None`:

Reply:
I've instead just done it as `self.wrapper_field_names = options.field_names` and the None check is now further down